composer.json

```diff
@@ -29,7 +29,7 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ~8.0",
```

Comment:
Should be `~7.3 || ~8.0.0 || ~8.1.0`

`8.2.0` onwards are still the wild west :-) 

A `composer update nothing` is also needed to update the `composer.lock` signature.

Reply:
Ah yea that would make sense